calico-cloud/visibility/elastic/flow/aggregation.mdx

```diff
@@ -10,96 +10,63 @@ Configure flow log aggregation level to reduce log volume and costs.
-| Pod prefix                 | Allowed and denied traffic (default is allowed) | `FlowLogsFileAggregationKindForAllowed`<br />**0**, No aggregation <br />**1**, A source port on each node.<br />**2, (default)** Source ports, or are from the same ReplicaSet. <br />**3**, Destination and source ports, and are from the same ReplicateSet.                                        |
-| Source port                | Allowed and denied traffic                      | `FlowLogsFileAggregationKindForAllowed`<br/>`FlowLogsFileAggregationKindForDenied`<br />**0**, No aggregation<br />**1, (default)** Source port on each node.<br />**2**, Source ports, or are from the same ReplicaSet.<br />**3**, Destination and source ports, and are from the same ReplicateSet. |
+For allowed flows, the default aggregation level is 2, `AnyProcessInSamePodPrefix` and for denied flows the default aggregation level is 1,
+`AnyProcessInSameSourcePod`.
+
+The following table summarizes the aggregation levels by flow log traffic:
+
+| **Level** | **Name**                            | **Description**                                                   |
+|-----------|-------------------------------------|-------------------------------------------------------------------|
+| 0         |                                     | No aggregation                                                    |
+| 1         | AnyProcessInSameSourcePod           | Identity fields below source pod level are masked out. It means that if multiple processes or containers, within the same source pod, perform the same operation, the events are aggregated.       |
+| 2         | AnyProcessInSameSourcePodPrefix     | In addition to the above, source pod names are aggregated based on their shared prefixes. This means that flows, to the same destination, from pods within the same Deployment/ReplicaSet are aggregated together. |
+| 3         | AnyProcessInSamePodPrefix           | This level of aggregation builds on the previous two levels and also groups destination pod names based on their shared prefixes. |
+
+### Understanding aggregation level differences
+
+Here are examples of pod-to-pod flows, highlighting the differences between flow logs at various aggregation levels.
+
+The source port is usually ephemeral and does not convey useful information. By suppressing the source port, `AnyProcessInSameSourcePod` aggregation
+type minimizes the flow logs generated for traffic coming from different containers within the same pod and going to the same destination endpoint
+and port. The two flows originating from `client-a` without aggregation are combined into one. 
+
+In Kubernetes, ReplicaSets and StatefulSets can automatically create names for pods. For example, the pods `nginx-1` and `nginx-2` are created by the
```

Comment:
I think it's worth mentioning Deployments and DaemonSets here too.  In particular I would guess that Deployment is the most common usage pattern.

Reply:
(If you need an umbrella term for all of those, I believe it is "pod controller".)